README.md

```diff
@@ -33,7 +33,7 @@ The repository for this project is one method where you can monitor progress - w
 
 ## Embedding pipeline
 
-To run the Zenml embedding pipeline, the resources such as AKS and ACR have to provisioned. AKS is used running Chroma server on AKS and ACR is used to store Chroma server image. 
+To run the Zenml embedding pipeline, the resources such as AKS and ACR have to provisioned. AKS is used running Chroma server on AKS and ACR is used to store Chroma server image.
```

Comment:
To run the embedding pipeline, both AKS and ARC need to be provisioned. We use AKS to run the Chroma (vector database) service and ACT is used to host the Chroma server image.

Reply:
I'm confused, what are ARC and ACT